tcms/testcases/migrations/0006_merge_text_field_into_testcase_model.py

```diff
@@ -58,7 +58,7 @@ class Migration(migrations.Migration):
         ),
 
         # copy the data from the related model
-        migrations.RunPython(forward_copy_data),
+        migrations.RunPython(forward_copy_data, migrations.RunPython.noop),
```

Comment:
Note: going backwards this will leave the text fields empty. See https://github.com/kiwitcms/Kiwi/pull/1839/commits/3a6e7fe7bcfe5fbbd8057a346b74839f42c8d8ba for example how to handle this.